bftengine/include/bftengine/Replica.hpp

```diff
@@ -69,6 +69,22 @@ class IRequestsHandler {
                       uint32_t &outReplicaSpecificInfoSize,
                       concordUtils::SpanWrapper &parent_span) = 0;
 
+  struct ExecutionRequest {
+    uint16_t clientId;
+    uint64_t executionSequenceNum = 0;
+    uint8_t flags;
+    std::string request;
+    std::string outReply;
+    uint32_t outActualReplySize;
+    uint32_t outReplicaSpecificInfoSize;
+    int outExecutionStatus;
+    uint64_t requestSequenceNum = executionSequenceNum;
+  };
+
+  virtual void execute(std::deque<ExecutionRequest> &requestList,
```

Comment:
requestList -> requests (no need to specify the data structure in the name)

Reply:
Fixed